app/api/validations/organization.py

```diff
@@ -1,7 +1,9 @@
 from app import messages
 from app.utils.bitschema_utils import OrganizationStatus, Timezone, Zone, ProgramStatus
 from app.utils.validation_utils import is_email_valid, is_phone_valid
-    
+from app.database.models.bit_schema.program import ProgramModel
+from app.messages import PROGRAM_NAME_ALREADY_USED, PROGRAM_NAME_IS_MISSING
```

Comment:
No need to add this import, `messages` is already imported.

Reply:
Ahh I missed this, thanks @ApheleiaS

Reply:
Yeah, I missed that too, I used auto-import from the Pycharm...